### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by rufener**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,73 @@
...
+
+Run your server with vite.
+```
+vite
+```
+Open [http://localhost:5173/web/sop/#/login](http://localhost:5173/web/sop/#/login) in your browser. You should see the login page of your application.
+
+# Deployment
+## Backend
+In your back directory, make your backend a python pyckage by running the following:
```
Comment (mparkan) [inline README.md:64]: 1. Open a new terminal in the "back" directory
2. Create a new python virtual environnement by typing `python -m venv venv` in the terminal
3. Activate the python virtual environnement: `.\venv\Scripts\activate`
4. Run `pip install -r requirement.txt` to install the dependencies

File: README.md [hunk 2]
```diff
@@ -1,14 +1,73 @@
...
+```
+
+2. Install requirements.
+```
+pip install -r requirements.txt
+```
+
+3. Copy the file `.env.template` to `.env` and complete it with your configuration settings.
+
+4. Run your backend server.
```
Comment (maltaesousa) [inline README.md:32]:
```suggestion
4. Move to your backend server.
```

File: README.md (hunk 1 above)
Comment (maltaesousa) [inline README.md:64]:
```suggestion
Follow Installation steps.
```

File: README.md [hunk 3]
```diff
@@ -1,14 +1,73 @@
...
+vite
+```
+Open [http://localhost:5173/web/sop/#/login](http://localhost:5173/web/sop/#/login) in your browser. You should see the login page of your application.
+
+# Deployment
+## Backend
+In your back directory, make your backend a python pyckage by running the following:
+```
+pip install -e .
+```
```
Comment (maltaesousa) [inline README.md:67]:
```suggestion
```

**Review by maltaesousa — APPROVED** [review]

**Comment by rufener** [comment]
Merci !

**Pull request merged**